### PR TITLE
Updated node dependencies

### PR DESCRIPTION
**Pull request by Hyperkid123**
Its about time we update the webpack configuration we use. There are some packages (like `webpack-serve`) which ale no longer supported and we should migrate to supported alternatives.

Also the webpack config is split accross 8 different files. That is not necessary. There is also a lot of configuration that is not being used at all and should be deleted.

### CODE REVIEW

File: config/spandx.config.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-/*global module, process*/
+/*global process*/
```
Comment (lgalis) [inline config/spandx.config.js]: As we update this file, I think we should follow the same pattern that the template frontend starter api does and move this file to a new directory, profiles/local-frontend, like here: https://github.com/RedHatInsights/insights-frontend-starter-app/blob/master/profiles/local-frontend.js - just to keep this consistent across apps.

Comment (Hyperkid123) [reply config/spandx.config.js]: I will take a look after the tests are working properly. :+1:

**Comment by Hyperkid123** [comment]
@lgalis now includes changes from https://github.com/RedHatInsights/catalog-ui/pull/300

**Comment by lgalis** [comment]
@Hyperkid - that was by mistake- will fix

**Comment by Hyperkid123** [comment]
@lgalis thank you for the help.

**Comment by lgalis** [comment]
The platform is down - will merge after it is back up and can run a test.

**Pull request merged**